api/source/database/migrations/20220909221517_citations.ts

```diff
@@ -11,6 +11,7 @@ export async function up(knex: Knex): Promise<void> {
       table.string("id").primary().unique();
       table.string("citation_title").notNullable();
       table.text("citation_description").notNullable();
+      table.text("citation_document").defaultTo("");
       table.text("image");
```

Comment:
Are you sure the relationships are okay?

Reply:
Yeah yeah, la muñeca fea 

No, si está bien pues

Reply:
😂😂😂😂😂